backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientResolver.java

```diff
@@ -14,6 +19,7 @@
 public class PatientResolver implements GraphQLQueryResolver {
 
   @Autowired private PersonService ps;
+  @Autowired private OrganizationService os;
```

Comment:
Field injection is [discouraged](https://stackoverflow.com/questions/39890849/what-exactly-is-field-injection-and-how-to-avoid-it),  I recommend using a constructor or `@RequiredArgsConstructor`

Reply:
Might as well stop the bleeding at some point 🙂 See 62e3f0d.